src/managers/builtin/venvUtils.ts

```diff
@@ -483,26 +488,11 @@ export async function createPythonVenv(
     if (customize === undefined) {
         return;
     } else if (customize === false) {
-        sendTelemetryEvent(EventNames.VENV_CREATION, undefined, { creationType: 'quick' });
-        const installables = await getProjectInstallable(api, project ? [project] : undefined);
-        const allPackages = [];
-        allPackages.push(...(installables?.flatMap((i) => i.args ?? []) ?? []));
-        if (options.additionalPackages) {
-            allPackages.push(...options.additionalPackages);
-        }
-        return await createWithProgress(
-            nativeFinder,
-            api,
-            log,
-            manager,
-            sortedEnvs[0],
-            venvRoot,
-            path.join(venvRoot.fsPath, '.venv'),
-            { install: allPackages, uninstall: [] },
-        );
+        return quickCreateVenv(nativeFinder, api, log, manager, sortedEnvs[0], venvRoot, options.additionalPackages);
```

Comment:
@karthiknadig just want to confirm this is good here to call the other command- seems intuitive to me but wasn't sure if there was a reason these were left seperate

Reply:
yep confirmed that works